CHANGELOG.md

```diff
@@ -3,47 +3,57 @@
 ### IMPROVEMENTS:
 * builder/amazon: Add no_ephemeral template option to remove ephemeral drives
     from launch mappings. [GH-8393]
-* builder/amazon: Add validation for `subnet_id` when specifying `vpc_id`
+* builder/amazon: Add validation for "subnet_id" when specifying "vpc_id"
     [GH-8360] [GH-8387] [GH-8391]
 * builder/amazon: allow enabling ena/sr-iov on ebssurrogate spot instances
     [GH-8397]
 * builder/amazon: Retry runinstances aws api call to mitigate throttling
     [GH-8342]
+* builder/hyperone: Update builder schema and tags [GH-8444]
 * builder/qemu: Add display template option for qemu. [GH-7676]
+* builder/qemu: Add fixer to convert disk size from int to string [GH-8390]
 * builder/qemu: When a user adds a new drive in qemuargs, process it to make
     sure that necessary settings are applied to that drive. [GH-8380]
 * builder/vmware: Fix error message when ovftool is missing [GH-8371]
+* core: Cleanup logging for external plugins [GH-8471]
 * core: Various fixes to error handling. [GH-8343] [GH-8333] [GH-8316]
     [GH-8354] [GH-8361] [GH-8363] [GH-8370]
+* post-processor/digitalocean-import: Drop unused test functions [GH-8474]
```

Comment:
Since this is a no-op change, it doesn't need to get included in the changelog. Pruning dead code doesn't affect users, and therefore mentioning it just adds noise.

Reply:
:+1: